dev-docs/source/environment.rst

```diff
@@ -47,6 +47,15 @@ This can be changed using::
     DSO_API_LOG_LEVEL = ...
     DSO_API_AUDIT_LOG_LEVEL = ...
 
+For some endpoints, when error messages are generated, those message
+are containing too much confidential information (like BSN numbers) to end up in the Sentry logs.
+
+So, it is possible to blacklist certain endpoints based on the `path` part of the url
```

Comment:
So, it is possible to ignore certain endpoints based on the `path` part of the url

Reply:
Oh yeah, good point. Forbidden word has sneaked into the documentation :-)